aesara/link/numba/dispatch/basic.py

```diff
@@ -636,43 +636,48 @@ def advancedincsubtensor1(x, vals, idxs):
         return advancedincsubtensor1
 
 
+@numba_njit(inline="always")
+def deepcopyop_1(x):
+    return x
+
+
+@numba_njit(inline="always")
+def deepcopyop_2(x):
+    return x.copy()
```

Comment:
FYI: If we remove the `numba_njit` decorators from here and apply them in `numba_funcify_DeepCopyOp`, the error should go away.

We should probably proceed in that fashion anyway, since some of the options specified in `numba_njit` are user-configurable within the Python session, and I'm not entirely sure if such changes will be affected if we AOT-like compile them in the Aesara package like this.

Reply:
I see. Then we will need to figure out some other way to cache and if the user changes configurations at every run, I think the previous approach would do better.